src/tests/integration-tests/scaling/horizontal_pod_autoscaling_test.go

```diff
@@ -11,6 +12,7 @@ import (
 
 var _ = Describe("Horizontal Pod Autoscaling", func() {
 	It("scales the pods accordingly", func() {
+		HPATimeout := MustHaveEnv("HPA_TIMEOUT")
```

Comment:
The environment variable should be optional, and go should default to a reasonable value.

Reply:
Added the default value which defaults what you used to have before.